core/domain/takeout_commit_message_truncate_jobs_one_off.py

```diff
@@ -0,0 +1,130 @@
+# coding: utf-8
```

Comment:
I think it would be okay to place these jobs to _activity_jobs_one_off.py_ instead of creating a new file.

Reply:
Is it alright if I leave it as a separate file? To me, this makes more sense since it matches up with how I did the last takeout one off jobs as well.

Reply:
I would agree with @vojtechjelinek here, actually -- we don't want a proliferation of job files all over the place, and this functionality isn't specifically related to takeout (it's more tied to the models that it is modifying).

Reply:
done